canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -50,7 +50,7 @@
 		.properties-radioset-tooltip {
 			display: flex;
 			align-items: center;
-			margin: 2.5px 0 1.5px 1.5px;
+			padding-left: 0;
```

Comment:
This doesnt seem to do anything when I tested in the browser. Did you see my comment here https://github.com/elyra-ai/canvas/pull/1277#discussion_r1034136022?

Reply:
Yes I did. What did understand from it is to replace `margin: 2.5px 0 1.5px 0` with `padding-left: 0`. correct me if im wrong

Reply:
That comment was saying to replace `margin: 2.5px 0 1.5px 1.5px;` with `margin: 2.5px 0 1.5px 0;`